test/NotificationActivationTests/APITests.cpp

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
```

Comment:
Same as previous comment and applies to all new copyright notices.
// Copyright (c) Microsoft Corporation and Contributors.
// Licensed under the MIT License.